pkg/ingester/user_state.go

```diff
@@ -30,6 +31,8 @@ type userStates struct {
 	limiter *Limiter
 	cfg     Config
 	metrics *ingesterMetrics
+	// Is tracking of active series enabled?
+	activeSeriesEnabled bool
```

Comment:
Food for thought: tor simplicity, what if we remove this and initialise `*ActiveSeries` (will be never filled) and we just don't register the gauge metric to the registerer?

Reply:
I turned that option down at first as I expected lot of tests calling `newIngesterMetrics`. Turns out it was fewer places, so I've changed code to do that instead.